### PR TITLE
Bug/disabled

### PR DESCRIPTION
**Pull request by ZhengHe-MD**

### CODE REVIEW

File: src/MDButton.jsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ class MDButton extends Component {
 }
 
 MDButton.propTypes = {
-  children: PropTypes.object,
+  children: PropTypes.any,
```
Comment (jimexist) [inline src/MDButton.jsx]: `component`?

Comment (jimexist) [inline src/MDButton.jsx]: or it might be text?

File: src/MDButton.jsx [hunk 2]
```diff
@@ -52,6 +53,7 @@ MDButton.propTypes = {
   isDisabled: PropTypes.bool,
   href: PropTypes.string,
   theme: PropTypes.string,
+  type: PropTypes.string,
```
Comment (jimexist) [inline src/MDButton.jsx]: this can be `enum`

File: src/MDButton.jsx (hunk 1 above)
Comment (ZhengHe-MD) [inline src/MDButton.jsx]: It can be PropTypes.element or PropTypes.text.. 
Maybe I should use PropTypes.oneOfType([PropTypes.element, PropTypes.string])

Comment (ZhengHe-MD) [inline src/MDButton.jsx]: It can be PropTypes.element or PropTypes.text.. 
Maybe I should use PropTypes.oneOf([PropTypes.element, PropTypes.text])

**Comment by jimexist** [comment]
@ZhengHe-MD still i see there is conflict - maybe should be fixing this before taking a look a CI?

**Pull request merged**